pandas/core/common.py

```diff
@@ -430,3 +433,83 @@ def _pipe(obj, func, *args, **kwargs):
         return func(*args, **kwargs)
     else:
         return func(obj, *args, **kwargs)
+
+
+def ensure_integer_dtype(arr, value):
```

Comment:
woa, what are you adding all of this code here for? its not the right place (don't add anything to core.common). can you make this much simpler. this is a very narrow case.